middle_end/flambda/types/env/aliases.ml

```diff
@@ -595,3 +621,29 @@ let get_canonical_ignoring_name_mode t name =
   match canonical t simple with
   | Is_canonical _ -> simple
   | Alias_of_canonical { canonical_element; _ } -> canonical_element
+
+let clean_for_export
+      { canonical_elements;
+        aliases_of_canonical_elements;
+        binding_times_and_modes; } =
+  let binding_times_and_modes =
+    Simple.Map.mapi (fun simple binding_time_and_mode ->
+        let module BTM = Binding_time.With_name_mode in
+        let new_mode =
+          if Simple.is_var simple then Name_mode.in_types
+          else BTM.name_mode binding_time_and_mode
+        in
+        BTM.create (BTM.binding_time binding_time_and_mode) new_mode)
+      binding_times_and_modes
+  in
+  let aliases_of_canonical_elements =
+    (* Note: the relative order of the aliases and of their canonical element
+       will be unchanged, as it only compares the binding times.
```

Comment:
Please clarify what "it" refers to.

Reply:
The "it" was supposed to refer to the order, but I agree that it's not obvious. Would "as it only depends on the binding times" be clearer ?

Reply:
Yes, fine.

Reply:
I've updated the branch with this version (and without the printing changes) in the meantime, so you should be able to merge directly.